documentation/asciidoc/computers/raspberry-pi/bcm2711-bootloader.adoc

```diff
@@ -280,6 +284,69 @@ Previously this property was only checked by the `rpi-eeprom-update` script. How
 
 Default: `0`
 
+[[HTTP_HOST]]
+==== HTTP_HOST
+
+If an HTTP download is initiated, `boot.img` and `boot.sig` are downloaded from this server.
+
+Invalid host names will be ignored. They should only contain lower case alphanumeric characters and `-` or `.`.
+If `HTTP_HOST` is set then HTTPS is disabled and plain HTTP used instead.
+You can specify an IP address to avoid the need for a DNS lookup.
+Don`t include the HTTP scheme or any forward slashes in the hostname.
+
+Default: `fw-download-alias1.raspberrypi.com`
+
+[[HTTP_PORT]]
+==== HTTP_PORT
+
+You can change the port used for HTTP downloads. This is ignored if `HTTP_HOST` is not set.
+
+By default network install and HTTP boot mode use port `443` because HTTPS is enabled when using the default host `fw-download-alias1.raspberrypi.com`. If `HTTP_HOST` is changed then HTTPS is not supported and HTTP will be used instead, so port `80` becomes the default.
+
+[[HTTP_PATH]]
+==== HTTP_PATH
+
+The path used for HTTP downloads.
+
+The case of the path *is* significant.
+Use forward (Linux) slashes for the path separator.
+Leading and trailing forward slashes are not required.
+
+If `HTTP_HOST` is not set, `HTTP_PATH` is ignored and the URL will be `\https://fw-download-alias1.raspberrypi.com:443/net_install/boot.img`. If `HTTP_HOST` is set the URL will be `\http://<HTTP_HOST>:<HTTP_PORT>/<HTTP_PATH>/boot.img`
```

Comment:
I don't think the back-slashes are needed before https and http here?

Reply:
I don't want these to be clickable as the user will see an invalid certificate error as we self sign the certs for these sites. This might worry them?

Reply:
and http://<HTTP_HOST>:<HTTP_PORT>/<HTTP_PATH>/boot.img is invalid

Reply:
Ah, I'd been assuming that links in `back-ticks` would remain as "just text", but I've just done some local testing and seen that links inside backticks _do_ get linkified by AsciiDoctor. So you were right, sorry!